test/remote/gateways/remote_adyen_test.rb

```diff
@@ -504,13 +531,15 @@ def test_missing_state_for_purchase
   def test_blank_country_for_purchase
     @options[:billing_address][:country] = ''
     response = @gateway.authorize(@amount, @credit_card, @options)
-    assert_success response
+    assert_failure response
+    assert_match Gateway::STANDARD_ERROR_CODE[:incorrect_address], response.error_code
```

Comment:
Do empty strings now send an empty string, or an `'N/A'`?

Reply:
(and should that behavior change, rather than the test?)

Reply:
Well, I guess it doesn't matter if the real behavior change is in what the gateway requires, which seems to be the case.

Reply:
empty strings raise an error. Not sending a values sends a `'N/A'`

Reply:
This returns the test to the state it was before the upgrade.

Reply:
> This returns the test to the state it was before the upgrade.

Makes sense that these tests were updated to be incorrect. When I did the upgrade it changed to succeeded because the billing address was no longer being sent where it was expected and thus none of it was present and there was nothing to error on. In hindsight that should've indicated that I needed to double check any changes to where the `billingAddress` hash was added.

Reply:
👍s all around.